libs/libskynet/src-test/skylinkvalidate.test.ts

```diff
@@ -0,0 +1,27 @@
+import { validateSkyfilePath } from "../src/skylinkvalidate.js";
+
+test("testValidateSkyfilePath", () => {
+  let tests = [
+    { trial: "test", result: true },
+    { trial: "test/subtrial", result: true },
+    { trial: "test/subtrial.ext", result: true },
+    { trial: "test/trial.ext/subtrial.ext", result: true },
```

Comment:
I would add `.foo` and `.foo/.bar` cases (contain/starts with dot but are valid paths)